src/DurableSDK/IPowerShellServices.cs

```diff
@@ -5,17 +5,26 @@
 
 namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
 {
+    using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
     using System;
     using System.Management.Automation;
 
     internal interface IPowerShellServices
     {
+        PowerShell GetPowerShell();
+
+        bool UseExternalDurableSDK();
```

Comment:
note to self: I think this method can be renamed to something like "HasExternalDurableSDK()"